project_task_description_template/data/demo.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="template1" model="project.task.template">
```

Comment:
**nitpick:** Please suffix all demo record XML IDs with demo, as specified in the docs.

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#demo-records

Reply:
Thanks, I'll fix it